scripts/common_functions.py

```diff
@@ -1,28 +1,11 @@
 import numpy as np
```

Comment:
Please put a docstring at the top of every file to describe the contents. It doesn't have to be extremely long, but it should orient the reader to the purpose of the code.